cmd/kubelet/app/options/options.go

```diff
@@ -133,6 +133,11 @@ type KubeletFlags struct {
 	RegisterSchedulable bool
 	// SeccompDefault enables the use of `RuntimeDefault` as the default seccomp profile for all workloads on the node.
 	SeccompDefault bool
+	// DisableCgroupV1Support prevents the kubelet from starting on hosts
+	// that use cgroup v1. By default, this is set to 'false', meaning
+	// the kubelet is allowed to start on cgroup v1 hosts unless this
+	// option is explicitly enabled.
+	DisableCgroupV1Support bool
```

Comment:
WDYT about following the convention of swap and call this `FailCgroupV1`?  technically we're not disabling but exiting if cgroupv1 is running.